contracts/contracts/RaidenMicroTransferChannels.sol

```diff
@@ -1,4 +1,4 @@
-pragma solidity ^0.4.17;
+pragma solidity ^0.4.19;
```

Comment:
are you using some 0.4.18/0.4.19 only features? If not then I guess this can stay as is.

Reply:
No, `0.4.17` should work too. I usually update this too keep up with https://etherscan.io/solcbuginfo . There is a fixed bug in 0.4.18 but it is of very low severity, though.

Reply:
These info appear on the verified contract, depending on the version you compiled the contract with and not the minimum required version for compiling shown in the source code.